openedx/core/djangoapps/credit/urls.py

```diff
@@ -19,4 +19,10 @@
         credit_provider_callback,
         name="provider_callback"
     ),
+
+    url(
+        r"^v1/provider/info/(?P<provider_id>[^/]+)/?$",
```

Comment:
This should simply be `v1/providers/(?P<provider_id>[^/]+)/?$`. Ideally *all* of the API URLs should use the plural form, providers, instead of provider.